packages/ui-react/src/components/PersonalDetailsForm/PersonalDetailsForm.tsx

```diff
@@ -44,7 +44,12 @@ const PersonalDetailsForm: React.FC<Props> = ({
   questionErrors,
 }: Props) => {
   const { t } = useTranslation('account');
-  const renderQuestion = ({ value, key, question, required }: CleengCaptureQuestionField) => {
+  const renderQuestion = ({ value, key, question, required, enabled }: CleengCaptureQuestionField) => {
+    // the question is not enabled, we don't need to render it
```

Comment:
I've seen your description where you note that you've added comments whilst the code speaks for itself, but I still have the question: what makes you think this code comment is necessary. In this instance, the code really speaks for itself, right? 😅

Reply:
You're totally right!

Reply:
Fixed 😄